docker-compose.override.yaml

```diff
@@ -49,8 +55,13 @@ services:
     volumes:
       - ./django:/app/django
     depends_on:
-      - postgresql
-      - rabbitmq
-      - minio
-      - redis-master
-      - django
+      django:
+        condition: service_healthy
+      postgresql:
+        condition: service_started
+      minio:
+        condition: service_started
+      redis-master:
+        condition: service_started
+      rabbitmq:
+        condition: service_started
```

Comment:
These will have to be rebased on the changes from #192.

Reply:
I thought I already merged those changes in this morning before I did this?